### ISSUE TITLE
Missing first option when using the <select multiple>

### ISSUE THREAD

**Issue by kinocz** [lekoala/bootstrap5-tags#116]
Hi,
I'm not a JS/HTML guru, so I'm not sure how to adequately describe the problem...

Even when sending all `option`s as `selected`, the first is missing from the tags.

I have created an example that is as simple as possible to mimic the problem.

https://jsfiddle.net/nkxutpv7/

**Comment by lekoala** [comment]
yes that's basically because typically the first option would be "please select..." something
if you don't provide a placeholder or something, it's going to take the first option as the placeholder
